### DISCUSSION TITLE
[anycable-go] Support for StatsD instrumentation

### DISCUSSION THREAD

**Discussion by daande** [anycable/anycable#114] [Ideas]
Any plans to add statsd support?

**Comment by palkan** [comment]
(@daande Moved this question to the Discussions (hope you don't mind 😉). Let's try this new way to communicate on GitHub.)

We don't have such plans yet, but that's something we might want to consider.

Ideally, I was hoping to have only one, _canonical_, way to provide instrumentation data. And we chose Prometheus.

Isn't there a way/tool to pull Prometheus metrics and push them into Statsd? For example, using [telegraf](https://github.com/influxdata/telegraf) or something like that.

**Reply by daande** [reply]
@palkan One of my team members (@jeremycw) put up this PR https://github.com/anycable/anycable-go/pull/126 thoughts?
